cve_bin_tool/cvedb.py

```diff
@@ -252,7 +252,9 @@ async def refresh(self):
             os.makedirs(self.cachedir)
         # check for the latest version
         if self.version_check:
-            self.LOGGER.info("Checking if there is a newer version.")
+            self.LOGGER.info(
+                f"cve-bin-tool is running {VERSION}. Checking if there is a newer version."
```

Comment:
Let's put 
```
    self.LOGGER.info(f"CVE Binary Tool version {VERSION}")
```
above line 254 so it prints the version even if the version check is disabled.  We might want to move it up so that it's the first thing printed when cli is run.  I think it's useful information to have in the console output.

I'm not sure we really need to report that were doing the check at all, only if it discovers that there's a newer version.  Do we need to async check_latest_version so that the tool doesn't pause here?  (the check is barely perceptable on my machine, but I have a fairly fast internet connection so it's possible that it's more noticeable to others).

Reply:
It works pretty fast for me too (my network speed is sometimes in kbps). I think we might not need it.